### DISCUSSION TITLE
How are you handling missing events from reconnecting a subscription?

### DISCUSSION THREAD

**Discussion by theogravity** [enisdenjo/graphql-ws#322] [Q&A] [answered]
Anyone have any advice on how you deal with events being lost during a period of disconnect to reconnect? One idea would be to track an id for each subscription, then on reconnect pass that back and have the pubsub service re-publish from that offset. Does anyone know if the Apollo client or `graphql-ws` handle anything relating to this under the hood automatically?

**Comment by enisdenjo** [comment, marked as answer]
Possible duplicate of https://github.com/enisdenjo/graphql-ws/discussions/175.
